README.md

```diff
@@ -91,18 +91,33 @@ Open JupyterLab: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybind
 
 ## Development
 
+First, set up the environment:
+
 ```shell
-mamba create -c conda-forge -y -n gator python jupyterlab=3
+mamba create -c conda-forge -y -n gator python jupyterlab
 mamba install -c conda-forge -y -n gator --file requirements_dev.txt
 conda activate gator
-pip install -e .
-jupyter server extension enable mamba_gator --sys-prefix
+```
 
+Clone the repo, install it with `pip`, and then watch for changes to the
+javascript:
+
+```bash
+git clone git@github.com:quansight/gator
+cd gator
+pip install -e .
 yarn install
```

Comment:
I think it might be good to leave the original one time build instructions and add the additional ones below for watching?

Reply:
Got it. I've added the original back.